Spigot-Server-Patches/0372-Fix-an-issue-where-the-vehicle-doesn-t-track-the-pas.patch

```diff
@@ -0,0 +1,45 @@
+From 7419c77daafd43dd22ded1c2271d9cd75a2ff2e9 Mon Sep 17 00:00:00 2001
+From: connorhartley <vectrixu+gh@gmail.com>
+Date: Mon, 31 Dec 2018 18:54:23 +1300
+Subject: [PATCH] Fix an issue where the vehicle doesn't track the passenger
+ when they disconnect
+
+
+diff --git a/src/main/java/net/minecraft/server/Entity.java b/src/main/java/net/minecraft/server/Entity.java
+index d44c55d8..476ceb71 100644
+--- a/src/main/java/net/minecraft/server/Entity.java
++++ b/src/main/java/net/minecraft/server/Entity.java
+@@ -86,7 +86,11 @@ public abstract class Entity implements ICommandListener, KeyedObject { // Paper
+     public boolean i; public boolean blocksEntitySpawning() { return i; } // Paper - OBFHELPER
+     public final List<Entity> passengers;
+     protected int j;
+-    private Entity au;public void setVehicle(Entity entity) { this.au = entity; } // Paper // OBFHELPER
++    // Paper start - OBFHELPER
++    public Entity getVehicleDirect() { return this.au; }
```

Comment:
`public Entity getVehicleDirect() { return this.au; } // Paper - OBFHELPER`

Reduces diff size

Reply:
I personally think having both of the getter and setter methods on a separate line is more readable. If more agree with you though, I can change it.

Reply:
It's to ensure a diff occurs if the field changes.

Reply:
readable yes, but same line guarantees a conflict. Sometimes our patch apply script can 'assume' it knows how to fix it, and it being 2 lines away is likely enough to trigger that false positive.

that's why I prefer to keep them same line, even with it being ugly, to ensure maintenance guarantees. Or least dont use the Paper Start line, just tag each and ensure its directly the next line that references it.